src/Sarif.Multitool.FunctionalTests/Rules/UrisMustBeValidTests.cs

```diff
@@ -19,13 +19,20 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
     public class UrisMustBeValidTests : ValidationSkimmerTestsBase<UrisMustBeValid>
     {
         [Fact(DisplayName = nameof(UrisMustBeValid_ReportsInvalidSarif))]
-        public void UrisMustBeValid_ReportsInvalidSarif() =>
+        public void UrisMustBeValid_ReportsInvalidSarif()
+        {
             // We need to disable compatibility transformations for any files that require
             // a malformed schema or malformed JSON, as this code fixes those things up
+            //
+            // As a result, when this test fails, you must open the test file below
+            // and manually bring it into conformance with the current v2 spec.
             Verify("Invalid.sarif", disablePrereleaseCompatibilityTransform: true);
+        }
```

Comment:
>} [](start = 8, length = 1)

*sigh* Your code base, I'm just the hired help ;-)

Reply:
hey! I only did this because the comments bothered me with no bracing. then i brought the single other test into conformance. you can revisit this point if you like. :) just revert it next time you're in here. i'm prepared to lose the argument.

---
In reply to: [255308227](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255308227) [](ancestors = 255308227)